general/administration/installing.md

```diff
@@ -37,142 +33,89 @@ The Jellyfin project and its contributors offer a number of pre-built binary pac
     - [Ubuntu Repository](#ubuntu-repository)
     - [Ubuntu Packages](#ubuntu-packages)
 
-## Containers
+## Container images
 
-[Open Container Initiative](https://stackoverflow.com/questions/31213126/libcontainer-vs-docker-vs-ocf-vs-runc#31219102) is a form of virtualization. An image is similar to an ISO while a container is akin to a running VM.
```

Comment:
I added this in here because I have people hit me up in the chat all the time.

Reply:
I understand due to the nature of this project many users can be inexperienced. Let's try to improve on this wording then, because it's factually incorrect.

Reply:
I try haha